fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/system/ChunkWriter.java

```diff
@@ -200,4 +201,11 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
             }
         }
     }
+
+    @Override
+    public void open(Serializable checkpoint) throws Exception
+    {
+        isExportPublic = FHIRConfigHelper.getBooleanProperty(FHIRConfiguration.PROPERTY_BULKDATA_BATCHJOB_ISEXPORTPUBLIC, true);
```

Comment:
WE could pass this in as a part of the job parameters

Reply:
I thought about this also and then finally wanted to make it simpler to allow the the job pick up the configuration by itself. we already have so many job parameters. :)